cmd/manager/main.go

```diff
@@ -159,6 +155,7 @@ func newRootCommand() *cobra.Command {
 			run := func(ctx context.Context) {
 				// Create a new Cmd to provide shared dependencies and start components
 				mgr, err := manager.New(cfg, manager.Options{
+					Scheme:             testfake.GetScheme(),
```

Comment:
This is the right move. As discussed, we need the package not to have `test` or `fake` in the name when we go live.